redbot/core/core_commands.py

```diff
@@ -1302,7 +1302,7 @@ async def serverprefix(self, ctx: commands.Context, *prefixes: str):
 
     @_set.command()
     @checks.is_owner()
-    async def locale(self, ctx: commands.Context, language_code: str):
+    async def globallocale(self, ctx: commands.Context, language_code: str):
         """
         Changes bot's locale.
```

Comment:
Should be changed to something like "Changes the default locale".

Reply:
Solved in https://github.com/Cog-Creators/Red-DiscordBot/pull/3896/commits/2210270f0bacd2556a566c0cd9b871eb24aef3ec